src/main/assembly/assembly-zip-kie-wb-playground.xml

```diff
@@ -0,0 +1,34 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+  <id>project</id>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <fileSets>
+
+    <fileSet>
+      <useDefaultExcludes>false</useDefaultExcludes>
+      <includes>
+        <include>**/pom.xml</include>
+      </includes>
+      <directory>target/kie-wb-playground-xslt</directory>
+      <outputDirectory>/</outputDirectory>
+    </fileSet>
+    <fileSet>
+      <useDefaultExcludes>false</useDefaultExcludes>
+      <excludes>
+        <exclude>**/*.iml</exclude>
+        <exclude>.idea/**</exclude>
+        <exclude>.git/**</exclude>
+        <exclude>.gitignore</exclude>
+        <exclude>LICENSE*</exclude>
+        <exclude>README.md</exclude>
+        <exclude>src/**</exclude>
+        <exclude>**/target/**</exclude>
+        <exclude>**/pom.xml</exclude>
```

Comment:
what if an example has a resource called mortagages/src/main/resources/org/foo/pom.xml ?
Might be better to use more restrictive excludes, so pom.xml and */pom.xml if possible.